src/fal/logger.py

```diff
@@ -0,0 +1,119 @@
+from dataclasses import dataclass
+
+from dbt.logger import log_manager
+from dbt.events.functions import fire_event
+from dbt.events.base_types import (
+    TestLevel as _TestLevel,
+    DebugLevel as _DebugLevel,
+    InfoLevel as _InfoLevel,
+    WarnLevel as _WarnLevel,
+    ErrorLevel as _ErrorLevel,
+)
+
+
+class LOGGER:
+    @classmethod
+    def test(cls, msg: str, *args, **kwargs):
+        fire_event(TestMessage(_prepare_msg(msg, *args, **kwargs)))
+
+    @classmethod
```

Comment:
WDYT about making `Logger` a normal class and having a single instance like `logger = Loggger()` at the end of the file (or `LOGGER`)? We could even have different `Logger` classes and having it configurable seems like a win (e.g. `logger` would now point to the `IPCLogger()`)., e.g. a logger class that uses a `fire_event_to_the_bridge` method etc.

Reply:
This is a valid approach too. We just don't really hold anything in the class.